packages/viz/test/utils/data.test.js

```diff
@@ -569,8 +569,16 @@ describe("DataUtil", () => {
       filterAllSpy.restore();
     });
 
-    it("should set the `days` property based on the endpoint range", () => {
-      dataUtil = new DataUtil(smbgData, opts({ endpoints: twoWeekEndpoints, chartPrefs: {} }));
+    it.only("should set the `days` property based on the endpoint range", () => {
+      dataUtil = new DataUtil(smbgData, opts({ endpoints: twoWeekEndpoints, chartPrefs: {activeDays: {
+        monday: true,
```

Comment:
code format has to be incorrect but I guess that we are in viz :cowboy_hat_face: